src/services/auth/plugins/password/utils.ts

```diff
@@ -49,6 +29,16 @@ export async function verifyCurrentPassword(
   return true;
 }
 
+/**
+ * Compares a plain password with a hash.
+ * @param password Password to encrypt.
+ * @param hash Hash to be compared against.
+ * @returns A promise to be either resolved with the comparison result salt or rejected with an Error
```

Comment:
I'm wondering if the promise is really rejected if the password doesn't correspond to the given hash ? It is not resolved with true or false ?